docs/deploying-helium.md

```diff
@@ -281,20 +308,20 @@ It is finally time to build Helium and then push the container image to the Azur
 2. Login the Docker CLI to your ACR:
 
 ```bash
-$ az acr login --name {app_prefix}heliumacr
+$ az acr login -n $acrName
 Login Succeeded
 ```
 
 3. Build / Docker-ize Helium:
 
 ```bash
-$ docker build --target=release -t {app_prefix}heliumacr.azurecr.io/helium:canary .
+$ docker build --target=release -t ${acrName}.azurecr.io/helium:canary .
 ```
 
 4. Push the Helium container image to the ACR:
```

Comment:
Use az acr build instead?

Reply:
Using ACR build would be simpler for the beginner and doesn't require any software beyond AZ CLI.

Reply:
Would this be the right replacement for the docker commands?: 
az acr build -t ${acrName}.azurecr.io/helium:canary --registry $acrName --image helium .